kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -334,16 +335,14 @@ def foundry_prove(
             method = [m for m in contract.methods if m.name == method_name][0]
             empty_config = foundry.kevm.definition.empty_config(GENERATED_TOP_CELL)
             kcfg = _method_to_cfg(empty_config, contract, method)
-            init_term = kcfg.get_unique_init().cterm.kast
-            target_term = kcfg.get_unique_target().cterm.kast
             _LOGGER.info(f'Expanding macros in initial state for test: {test}')
+            init_term = kcfg.get_unique_init().cterm.kast
             init_term = KDefinition__expand_macros(foundry.kevm.definition, init_term)
-            init_cterm = KEVM.add_invariant(CTerm(init_term))
```

Comment:
This call to `add_invariant` was happening _after_ macro expansion, and was adding terms like `#rangeUInt(...)` to the constraint. This call is removed, because the earlier call to `_method_to_cfg` already calls `add_invariant` for us anyway.